api/src/types/query.ts

```diff
@@ -10,8 +10,9 @@ export type Query = {
 	single?: boolean;
 	meta?: Meta[];
 	search?: string;
-	export?: 'json' | 'csv';
+	export?: 'json' | 'csv' | 'xliff' | 'xliff2';
 	deep?: Record<string, Query>;
+	optional?: Record<string, any>;
```

Comment:
What does this do?

Reply:
The 'optional' is an additional set of parameters that can be used by specific endpoints (and may be just in specific cases). So in order to not pollute type definition for Query with all this stuff I've added just one called 'optional' that may contain different props, depending on current endpoint